app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html

```diff
@@ -49,27 +49,29 @@
                             activity_status_change(index, event)
                         }
                         case EventType.CASE_CANCELLATION => {
-                            activity_case_cancellation(index, event)
+                            activity_case_cancellation(index, event, startAtTabIndex + (index*2))
                         }
                         case EventType.CASE_REFERRAL => {
-                            activity_case_referral(index, event)
+                            activity_case_referral(index, event, startAtTabIndex + (index*2))
                         }
                         case EventType.CASE_COMPLETED => {
-                            activity_case_completed(index, event)
+                            activity_case_completed(index, event, startAtTabIndex + (index*2))
                         }
                         case EventType.APPEAL_ADDED => {
                             val body = event.details.asInstanceOf[AppealAdded]
                             if(body.appealType == AppealType.REVIEW) {
                                 progressive_disclosure(
                                     id = s"activity-events-row-$index",
                                     summary = s"Review added with status ${AppealStatus.formatReview(body.appealStatus)}",
-                                    body = body.comment.map(Html(_))
+                                    body = body.comment.map(Html(_)),
+                                    Some(startAtTabIndex + (index*2))
```

Comment:
label "useTabIndex" param - lots of examples of this

Reply:
as above, will fix this